lnbits/core/helpers.py

```diff
@@ -58,14 +60,13 @@ async def stop_extension_background_work(ext_id: str, user: str):
     async with httpx.AsyncClient() as client:
         try:
             url = f"http://{settings.host}:{settings.port}/{ext_id}/api/v1?usr={user}"
```

Comment:
is this `?usr=` needed here?

Reply:
Good point. 
I've kept `usr` in case the install/uninstall endpoints are called via the API and no `access-token` is provided. One such scenario is the `lnbits-cli` calls.